### PR TITLE
Clarify what empty instrumentation name means, and missing instrumentation library

### PR DESCRIPTION
**Pull request by bogdandrutu**
Signed-off-by: Bogdan Drutu <bogdandrutu@gmail.com>

### CODE REVIEW

File: opentelemetry/proto/logs/v1/logs.proto [hunk 1]
```diff
@@ -26,106 +26,107 @@ option go_package = "github.com/open-telemetry/opentelemetry-proto/gen/go/logs/v
 
 // A collection of InstrumentationLibraryLogs from a Resource.
 message ResourceLogs {
-    // The resource for the logs in this message.
-    // If this field is not set then no resource info is known.
-    opentelemetry.proto.resource.v1.Resource resource = 1;
+  // The resource for the logs in this message.
+  // If this field is not set then no resource info is known.
```
Comment (yurishkuro) [inline opentelemetry/proto/logs/v1/logs.proto:30]:
```suggestion
  // If this field is not set then no resource info is unknown.
```

File: opentelemetry/proto/logs/v1/logs.proto [hunk 2]
```diff
@@ -26,106 +26,107 @@ option go_package = "github.com/open-telemetry/opentelemetry-proto/gen/go/logs/v
...
-    SEVERITY_NUMBER_WARN4  = 16;
-    SEVERITY_NUMBER_ERROR  = 17;
-    SEVERITY_NUMBER_ERROR2 = 18;
-    SEVERITY_NUMBER_ERROR3 = 19;
-    SEVERITY_NUMBER_ERROR4 = 20;
-    SEVERITY_NUMBER_FATAL  = 21;
-    SEVERITY_NUMBER_FATAL2 = 22;
-    SEVERITY_NUMBER_FATAL3 = 23;
-    SEVERITY_NUMBER_FATAL4 = 24;
+  // UNSPECIFIED is the default SeverityNumber, it MUST not be used.
```
Comment (yurishkuro) [inline opentelemetry/proto/logs/v1/logs.proto:50]:
```suggestion
  // UNSPECIFIED is the default SeverityNumber, it MUST NOT be used.
```

File: opentelemetry/proto/logs/v1/logs.proto [hunk 3]
```diff
@@ -27,7 +27,7 @@ option go_package = "github.com/open-telemetry/opentelemetry-proto/gen/go/logs/v
 // A collection of InstrumentationLibraryLogs from a Resource.
 message ResourceLogs {
   // The resource for the logs in this message.
-  // If this field is not set then no resource info is known.
+  // If this field is not set then no resource info is unknown.
```
Comment (Oberon00) [inline opentelemetry/proto/logs/v1/logs.proto:30]: Too many negations here: "If this field is **not** set then **no** resource info is **un**known.

Comment (yurishkuro) [reply opentelemetry/proto/logs/v1/logs.proto:30]: good catch
```suggestion
  // If this field is not set then resource info is unknown.
```

**Review by yurishkuro — APPROVED** [review]

File: opentelemetry/proto/common/v1/common.proto [hunk 4]
```diff
@@ -72,6 +72,8 @@ message StringKeyValue {
 // InstrumentationLibrary is a message representing the instrumentation library information
 // such as the fully qualified name and version. 
 message InstrumentationLibrary {
+  // An empty instrumentation library name is equivalent with an "unknown"
```
Comment (tigrannajaryan) [inline opentelemetry/proto/common/v1/common.proto:75]: I think this is confusing.It is not literally equivalent to to a library with the name equal to string "unknown", right? Can we remove the double quote characters and just say:
```suggestion
  // An empty instrumentation library name means the name is unknown. 
```

File: opentelemetry/proto/common/v1/common.proto [hunk 5]
```diff
@@ -72,6 +72,8 @@ message StringKeyValue {
 // InstrumentationLibrary is a message representing the instrumentation library information
 // such as the fully qualified name and version. 
 message InstrumentationLibrary {
+  // An empty instrumentation library name means the name is unknown. 
+  // instrumentation library.
```
Comment (tigrannajaryan) [inline opentelemetry/proto/common/v1/common.proto:76]: Line not needed anymore.

**Review by tigrannajaryan — APPROVED** [review]

File: opentelemetry/proto/common/v1/common.proto (hunk 5 above)
Comment (bogdandrutu) [reply opentelemetry/proto/common/v1/common.proto]: Done.

**Pull request merged**